docs/manual/input.md

```diff
@@ -1,3 +1,16 @@
+---
+title: Just say hello!
+author: My Friend
+header-includes: |
+    \usepackage{tikz,pgfplots}
+    \usepackage{fancyhdr}
+    \pagestyle{fancy}
+    \fancyhead[CO,CE]{This is fancy}
+    \fancyfoot[CO,CE]{So is this}
+    \fancyfoot[LE,RO]{\thepage}
+abstract: This is a pandoc test with Markdown + inline LaTeX
```

Comment:
why do we have this section?

Reply:
I removed it after discussion with @wschoonveld , @CindyvdVries  can put it back if it was really necessary